docs_source_files/content/driver_idiosyncrasies/driver_specific_capabilities.de.md

```diff
@@ -36,7 +36,14 @@ var driver = new FirefoxDriver(options);
 # We don't have a Ruby code sample yet -  Help us out and raise a PR
   {{< / code-panel >}}
   {{< code-panel language="javascript" >}}
-// We don't have a JavaScript code sample yet -  Help us out and raise a PR  
+import * as firefox from 'selenium-webdriver/firefox';
+import { Builder } from 'selenium-webdriver';
+
+const options = new firefox.Options();
```

Comment:
hi @natanportilho,

Like other code samples, can you set any firefox capability in the JS code sample

adding this line will do i think =>  "options.headless();"

Reply:
Hello, I've changed the code to use the options this way. Thank for your help.